x-pack/plugins/observability_solution/slo/server/lib/rules/slo_burn_rate/lib/summary_repository.ts

```diff
@@ -22,12 +22,9 @@ export async function getSloSummary(
         query: {
           bool: {
             filter: [
-              {
-                term: { 'slo.id': slo.id },
-              },
-              {
-                term: { 'slo.instanceId': instanceId },
-              },
+              { term: { 'slo.id': slo.id } },
+              { term: { 'slo.revision': slo.revision } },
```

Comment:
💬 Filtering on the slo.revision won't change much in most cases, but cannot hurt when the SLO is updated and the previous data is still available when the rule runs